static/app/components/group/issueSeerBadge.tsx

```diff
@@ -0,0 +1,61 @@
+import styled from '@emotion/styled';
+
+import {SeerIcon} from 'sentry/components/ai/SeerIcon';
+import {Tooltip} from 'sentry/components/core/tooltip';
+import {
+  getAutofixRunExists,
+  isIssueQuickFixable,
+} from 'sentry/components/events/autofix/utils';
+import Link from 'sentry/components/links/link';
+import {space} from 'sentry/styles/space';
+import type {Group} from 'sentry/types/group';
+import useOrganization from 'sentry/utils/useOrganization';
+
+interface IssueSeerBadgeProps {
+  group: Group;
+}
+
+function IssueSeerBadge({group}: IssueSeerBadgeProps) {
+  const organization = useOrganization();
+  const issuesPath = `/organizations/${organization.slug}/issues/`;
```

Comment:
prefer using `const {baseUrl} = useGroupDetailsRoute();` since the user could be visiting a specific event id and we don't want to lose that

Reply:
But this is in the issue stream, not issue details, right?